cocktails.web/.env.cypress

```
@@ -1,15 +1,17 @@
-VITE_B2C_POLICY=B2C_1_SignInSignUp_Policy
-VITE_B2C_RESET_PASSWORD_POLICY=B2C_1_ResetPassword_Policy
+
+VITE_AUTH0_DOMAIN=cezzis.us.auth0.com
+VITE_AUTH0_CLIENT_ID=HZXyiZxjHgkQqeb6UJxmikCGYpSz5iPb
+VITE_AUTH0_REDIRECT_URI=https://localhost:4001/
+VITE_AUTH0_COCKTAILS_API_AUDIENCE=https://cezzis-cocktails-api
+
 VITE_COCKTAILS_API_URL=https://localhost:7176
 VITE_COCKTAILS_IMAGE_URL=https://localhost:7176/api/v1/images
-VITE_RECAPTCHA_SITE_KEY=6Le3QR0qAAAAAIAUOYQO7P6KoFCVdZuAR1FwvUnG
 VITE_COCKTAILS_APIM_SUBSCRIPTION_KEY=383hudiudhUJK984jdus7HDY
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_COCKTAILS_APIM_SUBSCRIPTION_KEY key should go before the VITE_COCKTAILS_API_URL key

(UnorderedKey)